modules/manually-creating-alibaba-ram-user.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/manually-creating-alibaba-ram.adoc
+
+:_content-type: PROCEDURE
+[id="manually-creating-alibaba-ram-user_{context}"]
+= Creating the required RAM user
+
+// https://github.com/openshift/cloud-credential-operator/pull/412/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR232
+
+When you configure the Resource Access Management (RAM) user, be sure to consider the following requirements:
+
+* Attach the `AdministratorAccess` and `AliyunBSSFullAccess` policies to a RAM user to ensure that the account has sufficient permission to create the {product-title} cluster. This policy grants permissions to manage all Alibaba Cloud resources. The user must have the following permissions:
++
+** ram:CreatePolicy
+** ram:GetPolicy
+** ram:CreatePolicyVersion
+** ram:DeletePolicy
+** ram:DetachPolicyFromUser
+** ram:ListPoliciesForUser
+** ram:AttachPolicyToUser
+** ram:CreateUser
+** ram:GetUser
+** ram:DeleteUser
+** ram:CreateAccessKey
+** ram:ListAccessKeys
+** ram:DeleteAccessKey
+** bss:CreateInstance
+** bss:Query
+** bss:ModifyInstance
+
```

Comment:
@kwoodson @mtulio Can you verify if the addition of the AliyunBSSFullAccess` policy and the `bss:*` permissions is correct? Based on @gaurav-nelson [request in Slack](https://coreos.slack.com/archives/C01PES793LL/p1644336131581319).

Reply:
That's from @gauravsingh85

Reply:
@mburke5678 Ii will create a cluster today with a custom user with those credentials and give you feedback.

Reply:
I just install a cluster with a user with only one managed policy `AdministratorAccess`, and it was enough to create the RAM users and install the cluster - as expected it's one of the most permissive policies:
```json
{
    "Statement": [
        {
            "Action": "*",
            "Effect": "Allow",
            "Resource": "*"
        }
    ],
    "Version": "1"
}
```

```
time="2022-02-09T14:53:23-03:00" level=debug msg="Time elapsed per stage:"
time="2022-02-09T14:53:23-03:00" level=debug msg="           cluster: 2m56s"
time="2022-02-09T14:53:23-03:00" level=debug msg="         bootstrap: 1m16s"
time="2022-02-09T14:53:23-03:00" level=debug msg="Bootstrap Complete: 12m45s"
time="2022-02-09T14:53:23-03:00" level=debug msg="               API: 2m4s"
time="2022-02-09T14:53:23-03:00" level=debug msg=" Cluster Operators: 9m54s"
time="2022-02-09T14:53:23-03:00" level=info msg="Time elapsed: 27m0s"
```

Reply:
I have not tried cluster install but yes I believe the following permissions are not required. 
```
bss:CreateInstance
bss:Query
bss:ModifyInstance
```

Reply:
@mtulio Any luck with this ^^?

Reply:
@mtulio Do you have an environment I could use to try an install, with access to create RAM users and such?

Reply:
@mburke5678 sent!